packages/harpguru-core/src/hooks/use-flush-buffered-activity-toggles/use-flush-buffered-activity-toggles.ts

```diff
@@ -0,0 +1,133 @@
+import { useDispatch } from 'reactn'
+import { unstable_batchedUpdates } from 'react-native'
+import type { HarpStrataProps, HarpStrata } from 'harpstrata'
+import { getHarpStrata } from 'harpstrata'
+import type { DegreeIds } from 'harpparts'
+
+// TODO: obviously this is not the right place to import this from
+import type { GlobalState } from '../../types'
+import { batchToggleDegreeIds } from '../../components/toggle-buffer-flusher/utils'
+
+const flushBufferedActivityToggles = (
```

Comment:
This function along with it's recorded mystery will need to be deleted which is a shame in a way. I should record this message as an issue to be resolved, even if it's never relevant again and I never solve it. It's a very strange problem.

Reply:
Raised in https://github.com/js-jslog/harpguru/issues/61